packages/components/src/range-control/test/index.js

```diff
@@ -58,7 +57,7 @@ describe( 'RangeControl', () => {
 	} );
 
 	describe( 'validation', () => {
-		it( 'should not apply if new value is lower than minimum', () => {
+		it( 'should not apply new value is lower than minimum', () => {
```

Comment:
In this test and the below one you missed the `if` in description.
Also I think a better approach to tests is to test what is expected and not what is shouldn't be.

For example I think it should be like: `expect( rangeInput.value ).toBe(.......`

Reply:
Great suggestion! 🙏 . Will make that update

Reply:
Thank you! Fixed :D